feature/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go

```diff
@@ -24,11 +24,76 @@ import (
 	"github.com/openconfig/ondatra"
 	"github.com/openconfig/ondatra/gnmi"
 	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+type Testcase struct {
+	name string
+	fn   func(t *testing.T)
+}
+
+var (
+	QoSConfigTestcases = []Testcase{
+		{
+			name: "testQoSForwadingGroupsConfig",
+			fn:   testQoSForwadingGroupsConfig,
+		},
+		{
+			name: "testQoSClassifierConfig",
+			fn:   testQoSClassifierConfig,
+		},
+		{
+			name: "testQoSInputIntfClassifierConfig",
+			fn:   testQoSInputIntfClassifierConfig,
+		},
+		{
+			name: "testECNConfig",
+			fn:   testECNConfig,
+		},
+		{
+			name: "testSchedulerPoliciesConfig",
+			fn:   testSchedulerPoliciesConfig,
+		},
+		{
+			name: "testQoSOutputIntfConfig",
+			fn:   testQoSOutputIntfConfig,
+		},
+	}
+)
+var (
```

Comment:
You can put these two variables in the same var section.

Reply:
done